README.md

```diff
@@ -195,3 +195,40 @@ OUTPUT
 
 Now compile your maven project. If all went well, you should see 2 files inside
 the directory ${basedir}/target/generated-sources/java/com/example/db.
+
+### Freemarker
+To use a Freemarker Template. Create a folder anywhere: \$HOME/templates/Example
+Inside that folder create the freemarker template file: ExampleDB.py.ftl
+
+Jportal supports freemarker file name substitution. For example ExampleDB${database.tables[0].name}.py.ftl will generate a ExampleDB{tablename}.py file for all the SI files you gen for. 
+
+Inside your template file:
+```
+<#list database.tables as table>
+
+class T${table.name}(object):
+    def __init__(self<#list table.fields as field>, ${field.name?lower_case}=None</#list>):
```

Comment:
This syntax example is a bit complex, might be worthwhile to do a slightly simpler one in future.